invenio/modules/oauthclient/contrib/orcid.py

```diff
@@ -74,7 +74,10 @@
 """
 
 import copy
-from flask import session
+from flask import current_app, session
```

Comment:
just one more empty line before the `flask` import section if you want to be 100% stylish ;)